lib/Server.js

```diff
@@ -445,13 +445,22 @@ class Server {
     const compilerOptions = this.getCompilerOptions();
     // TODO remove `{}` after drop webpack v4 support
     const watchOptions = compilerOptions.watchOptions || {};
+    const defaultWatchOptions = {
+      ignoreInitial: true,
+      persistent: true,
+      followSymlinks: false,
+      atomic: false,
+      alwaysStat: true,
+      ignorePermissionErrors: true,
+      ...watchOptions,
+    };
```

Comment:
Let's move normalization from https://github.com/webpack/webpack-dev-server/blob/b4a0cfd326cfa73a4c932254244df44a3deec1a1/lib/Server.js#L2156 to here, wecan calculate default `usePolling` and `interval` value here